### PR TITLE
English translation for internet module

### PR DESCRIPTION
**Pull request by kiscs**
Translated internet module to English.

### CODE REVIEW

File: resources/lang/en/internet.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "delete" => "Delete",
+    "cancel" => "Cancel",
+    "confirm_delete" => "Are you sure you want to delete?",
+    'error' => 'Error',
+    'ajax_error' => 'Server error! Please refresh the site!',
+    'ok' => 'OK',
+    'username' => 'Username',
+    'approve' => 'Approve',
+    'reject' => 'Reject',
+    'request' => 'Undo',
```
Comment (luksan47) [inline resources/lang/en/internet.php]: This looks even worse in english. Maybe the key should be changed?
cc @vendi95

File: resources/lang/en/internet.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    'user_id' => 'User id',
+    'internet' => 'Internet',
+    'auto_approved_mac_slots_long' => 'Automatically approved devices: :count pcs',
+    'registered_devices_info' => 'You must register your device to use wired connection. You can approve a certain number of devices automatically. If you need more, then the administrators must approve it after your registration; you can provide a reason in the comment field.',
+    'edit_warning_auto_approve' => 'Due to automatic approvement after editing, the state might remain unchanged.',
+    'wifi_password_info' => 'You can connecnt to ejc_wifi with the credentials listed below. You can change your password to a randomly generated one, The administrators will do this regularly, but you will be notified in advance.',
```
Comment (luksan47) [inline resources/lang/en/internet.php]: > one, The

change comme to dot.

Comment (kiscs) [reply resources/lang/en/internet.php]: done.

File: resources/lang/en/internet.php (hunk 1 above)
Comment (steakhal) [reply resources/lang/en/internet.php]: What's going on? Request -> undo what?

Comment (steakhal) [reply resources/lang/en/internet.php]: ping @kiscs

**Pull request merged**